spec/controllers/api/v1/froggo_team_memberships_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::FroggoTeamMembershipsController, type: :controller do
+  let(:user) { create(:github_user, login: "user_login") }
+  let(:platanus_org) { create(:organization, login: "platanus", members: [user]) }
+  let!(:platanus_team) { create(:froggo_team, name: "test_name", organization: platanus_org) }
+  let(:buda_org) { create(:organization) }
+  let!(:buda_team) { create(:froggo_team, name: "test_name", organization: buda_org) }
+  let(:user_1) { create(:github_user, login: "user1_login") }
+  let!(:join) { create(:froggo_team_membership, github_user: user_1, froggo_team: platanus_team) }
+  let(:github_session) do
+    double(
+      token: "token",
+      organizations: [{ id: platanus_org.gh_id, role: "member" }],
+      user: user
+    )
+  end
+
+  before do
+    expect(subject).to receive(:authenticate_github_user).and_return(true)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.